docs/share-action.md

```diff
@@ -0,0 +1,137 @@
+# Draft Share Action
+
+## Specification
+
+||External (OData) Name|Internal (ABAP) Name|Realization|Comment|
+|---|---|---|---|---|
+|Action|Share|`SHARE`| ||
+|Action Parameter|ShareAll|`SHAREALL`|`ABP_BEHV_PAR_SHARE`||
+|Action Parameter|IsDeltaUpdate|`ISDELTAUPDATE`|`ABP_BEHV_PAR_SHARE`||
+|Action Parameter|Users|`USERS`|`ABP_BEHV_PAR_SHARE`|Table of `ABP_BEHV_DRAFT_USER_ACCESS`|
+|Complex Type|DraftUserAccessType||`ABP_BEHV_DRAFT_USER_ACCESS`||
+|Complex Type Property|UserID|`USERID`|||
+|Complex Type Property|UserAccessRole|`USERACCESSROLE`|||
+
+### Example
+
+```xml
+<ComplexType Name="DraftUserAccessType">
+  <Property Name="UserID" Type="Edm.String" Nullable="false" MaxLength="12"/>
+  <Property Name="UserAccessRole" Type="Edm.String" Nullable="false" MaxLength="1"/>
+</ComplexType>
+<Action Name="Share" IsBound="true">
+  <Parameter Name="_it" Type="com.sap.gateway.srvd.i_shareddemoaction_sd.v0001.SharedEntityType" Nullable="false"/>
+  <Parameter Name="ShareAll" Type="Edm.Boolean" Nullable="false">
+    <Annotation Term="Core.OptionalParameter">
+      <Record/>
+    </Annotation>
+  </Parameter>
+  <Parameter Name="IsDeltaUpdate" Type="Edm.Boolean" Nullable="false" DefaultValue="false"/>
+  <Parameter Name="Users" Type="Collection(com.sap.gateway.srvd.i_shareddemoaction_sd.v0001.DraftUserAccessType)" Nullable="false">
+    <Annotation Term="Core.OptionalParameter">
+      <Record/>
+    </Annotation>
+  </Parameter>
+</Action>
+```
+
+## Semantics
+
+ShareAll = true, IsDeltaUpdate = true, Users = $[]$ or $[C]$ (the current user)
```

Comment:
This list does not explain the effect of the `UserAccessRole` property in `Users`.